packages/components/src/placeholder/style.scss

```diff
@@ -148,11 +148,6 @@
 			padding: 0 $grid-unit-10 2px;
 		}
 	}
-	.components-placeholder__learn-more {
-		.components-external-link {
-			color: var(--wp-admin-theme-color);
-		}
-	}
```

Comment:
`.components-placeholder__learn-more` is only used in the Embed block in wp-block-library, so it doesn't belong here. Moved and renamed.

Reply:
Nice lceanup